assets/src/block-editor/components/sidebar-notification/style.css

```diff
@@ -0,0 +1,70 @@
+.sidebar-notification {
+	display: flex;
+	min-height: 67px;
+	overflow: hidden;
+	padding: 14px 18px;
+	position: relative;
+}
+
+.sidebar-notification.is-error::before {
+	background: #d64e21;
+	bottom: 0;
+	content: "";
+	left: 0;
+	position: absolute;
+	top: 0;
+	width: 5px;
+}
+
```

Comment:
```suggestion
```

![image](https://user-images.githubusercontent.com/134745/111580976-b1de6e80-8775-11eb-86c9-4cc03864ad6b.png)

Reply:
The entire `isError` component state can be removed since it's the only instance where it's used.

It's been done in 6b78c8b.